docs/resources/service_account_key.md

```diff
@@ -0,0 +1,46 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_service_account_key Resource - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Service Key authorizes access to all Resources assigned to the Service. For more information, see Twingate's documentation https://www.twingate.com/docs/services/#2-select-generate-key-to-create-a-new-service-key.
+---
+
+# twingate_service_account_key (Resource)
+
+Service Key authorizes access to all Resources assigned to the Service. For more information, see Twingate's [documentation](https://www.twingate.com/docs/services/#2-select-generate-key-to-create-a-new-service-key).
```

Comment:
s/Service Key/A Service Key/ at the start of the paragraph

s/the Service/a Service Account/

Reply:
updated